ui/src/app/views/workspace/component.ts

```diff
@@ -129,12 +135,27 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
       this.researchPurposeArray.slice(
         this.leftResearchPurposes.length,
         this.researchPurposeArray.length);
-    this.showTip = true;
+    this.showTip = false;
+    this.pageId = 'workspace';
+    const pageVisit: PageVisit = { page: this.pageId };
+    this.newPageVisit = pageVisit;
   }
 
   ngOnInit(): void {
     this.wsNamespace = this.route.snapshot.params['ns'];
     this.wsId = this.route.snapshot.params['wsid'];
+    this.profileService.getPageVisits().subscribe(
+      pageVisitsReceived => {
+        this.firstVisit = !pageVisitsReceived.some(v =>
+          v.page === this.pageId);
+      },
+      error => {
+        this.pageVisitsError = true;
+      },
+      () => {
+        if (this.firstVisit) { this.showTip = true; }
```

Comment:
nit: AFAIK our code almost always puts newlines here and uses brackets for if statements. Personally I'm also OK with 1-liners where the brackets are excluded, but I'm not sure if that particular style issue has ever been brought up with the group.